build/gen-deb.sh

```diff
@@ -0,0 +1,45 @@
+set -e
+
+if [ ! -f "opa_linux_amd64" ]; then
+  echo "ERROR: File 'opa_linux_amd64' not found. First build it 'make build-linux'."
+  exit 1
+fi
+
+if [ -z "$VERSION" ]; then
+  echo "ERROR: Need to have VERSION environment variable set."
+  exit 1
+fi
+
+rm -rf tmp_working_dir_opa
+mkdir tmp_working_dir_opa
+cd tmp_working_dir_opa
+mkdir -p ./usr/bin/
+cp ../opa_linux_amd64 ./usr/bin/opa
+chmod +x ./usr/bin/opa
+
+tar cJf data.tar.xz ./usr/bin/opa #./usr/share/man/<location> <- in gz format
```

Comment:
This comment reminds me, can we generate and install the opa man pages with the deb file too?

Reply:
done